recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
 {% set version = "0.6" %}
-{% set sha256 = "b4be2b3438971588dc306116658f475916755ad8fcc0618e79023846ce94f97b" %}
```

Comment:
Oh, just did not see much an advantage of adding an indirection for one occurrence here.
Changed and also moved `number` here so that all variable info can be edited at the top.